ci/gpu/build.sh

```diff
@@ -71,8 +71,8 @@ conda install "rmm=$MINOR_VERSION.*" "cudatoolkit=$CUDA_REL" \
               "ucx-py=${MINOR_VERSION}" \
 
 # https://docs.rapids.ai/maintainers/depmgmt/
-# conda remove -f rapids-build-env rapids-notebook-env
-# conda install "your-pkg=1.0.0"
+conda remove --force rapids-build-env rapids-notebook-env
+conda install "pandas>=1.0,<1.2"
```

Comment:
Revert these changes now that the upstream PR merged and we should be good to go here

Reply:
done.